DataFormats/PatCandidates/interface/IsolatedTrack.h

```diff
@@ -121,6 +121,17 @@ namespace pat {
 
     const reco::HitPattern& hitPattern() const { return hitPattern_; }
 
+    // helper functions for PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py
+    int numMissingInnerHits() const {
+      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::MISSING_INNER_HITS);
+    }
+    int numMissingMiddleHits() const {
+      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::TRACK_HITS);
+    }
+    int numMissingOuterHits() const {
+      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::MISSING_OUTER_HITS);
```

Comment:
these are quite confusingly not matching `TrackBase::missingInnerHits` implementation `hitPattern_.numberOfLostHits(HitPattern::MISSING_INNER_HITS)`

Looking in other places in cmssw, the shorthand for the `trackerLayersWithout` variables are `lostInnerLayers`, `lostLayers`, and `lostOuterLayers`, respectively

```suggestion
    int lostInnerLayers() const {
      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::MISSING_INNER_HITS);
    }
    int lostLayers() const {
      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::TRACK_HITS);
    }
    int lostOuterLayers() const {
      return hitPattern_.trackerLayersWithoutMeasurement(reco::HitPattern::MISSING_OUTER_HITS);
```